src/test/com/oop/RunPhone.java

```diff
@@ -0,0 +1,16 @@
+package test.com.oop;
+
+public class RunPhone {
+    public static void main(String[] args) {
+        Phone phone1 = new Phone("123456789", "Samsung", 0.3);
+        Phone phone2 = new Phone("987654321", "iPhone");
+        Phone phone3 = new Phone();
+
+
```

Comment:
2 empty lines can be used only in case you really se the necessity for it that you can explain. Otherwise one line is sufficient.